test/suites/ACID.m

```diff
@@ -2458,6 +2459,31 @@
   set(h, 'orientation', 'horizontal');
 end
 % =========================================================================
+function [stat] = pruneOutsideBox()
+    % test function 'pruneOutsideBox' that is part of 'cleanfigure'
+    stat.description = ['Test function \texttt{pruneOutsideBox()} ', ...
+                        'of \texttt{cleanfigure()}.'];
+    stat.issues      = 226;
+
+    % generate 16 deterministic values
+    data = magic(4);
+    data = data(:);
+
+    % create plots
+    subplot(2,1,1);
+    plot(data,'x-');
+    xlim([10 12]);      % only show 3 values
+    title('regular plot');
+
+    subplot(2,1,2);
+    stairs(data,'x-');
+    xlim([10 12]);      % only show 2 steps
+    title('stairs plot');
```

Comment:
I would avoid adding an additional test to the acid and instead edit the ACID(7) by:
* making it a subplot where the second plot is the `stairs...` (which in R2014b has its own type, otherwise is a `'line'`)
* add `stat.issues = 226;`

We already have too many tests in the acid, and many overlap of those overlap. This one tests the stairs type which was introduced in R2014b but otherwise is really not too different from ACID(7) to require a separate figure.